docs/extra_pages.dox

```diff
@@ -33,6 +33,52 @@
  * \page installation_instructions Installation instructions
  *  
  * \section requirements Requirements
+ * \subsection xraylib Xraylib
```

Comment:
Mss best dat je dit Requirements stuk eens nakijkt of het min of meer deftig of niet te uitvoerig is of whatever. Heb zo maar iets gedaan...

Reply:
En bij Usage etc. weet ik ook niet zo goed wat te schrijven... Ben niet helemaal mee of de users dan het volledige `polycap.h` moeten includen, of de deel-functies ervan, ...

Reply:
Hey Pieter, zal er volgende week eens naar kijken...

Reply:
Hey, ben terug op Diamond nu... zal proberen zo snel mogelijk naar alles te kijken maar zou wel eens een paar dagen kunnen duren. Sorry voor de vertraging!

Reply:
No problem. Doe maar opt gemak :) Ben ondertussen (zoals je vast gezien hebt) wat van de oude todo's aan het (proberen) oplossen.

Hoe was de conferentie verder nog? Veel succes gehad met je eigen posters? Toevallig nog iets gemerkt van interesse voor mijn/onze poster?

Reply:
Heb achteraf nog een paar mensen een demonstratie gegeven met dat python scriptje. Ze leken toch wel wat enthousiast.

Merci om nog wat aan de todo lijst te werken...

Reply:
👍 
en jij ook merci om het demonstratietje te geven :) Ik vermoed niet dat er grote namen tussen het publiek zaten (dat je weet, ikzelf ben miserabel slecht in weten wie welke persoon is, dus laat staan of het belangrijke mensen zijn of niet)